x-pack/plugin/esql/qa/testFixtures/src/main/resources/date.csv-spec

```diff
@@ -1930,3 +1930,101 @@ from employees | where emp_no == 10040 | eval x = day_name(birth_date) | keep em
 emp_no:integer | birth_date:date                     | hire_date:date             | x:keyword
 10040          | null                                | 1993-02-14T00:00:00.000Z   | null  
 ;
+
+
+monthNameRowTest
+required_capability:fn_month_name
+// tag::docsMonthName[]
+ROW dt = to_datetime("1996-03-21T00:00:00.000Z")
+| EVAL monthName = MONTH_NAME(dt);
+// end::docsMonthName[]
+
+dt:date   | monthName:keyword
+1996-03-21T00:00:00.000Z | March
```

Comment:
If you do `tag::docsMonthName-result` then you can include this result table - it gets rendered as an actual html table.